src/components/ViolenceTestCards/index.scss

```diff
@@ -0,0 +1,127 @@
+@import "../../style/variables";
+
+.card {
```

Comment:
be careful having a classname called .card here. maybe name it mre specifically. as i'm sure your teammates are also using things called card

Reply:
ok changed to .violenceTestCard